parsers/json_parser_listener.go

```diff
@@ -215,14 +215,17 @@ func (l *JsonParserListener) EnterNumber(ctx *parser_json.NumberContext) {
 
 	// Create holder to store the node where the location information
 	// of the number value should be stored
-	locationInfoDest := l.getTOS()
+	var locationInfoDest *Node
 	if l.configFile == nil { // This number is the top level entity
 		// Create new node for number
 		node := &Node{Type: numberType, Value: value}
 
 		// Set config file to point to this number
 		l.configFile = node
 
+		// Push object node to stack
+		l.stack.Push(node)
```

Comment:
Why this push?

Reply:
We only push to the stack nodes that contain other nodes: Objects, Pairs, Arrays